src/main/scala/org/broadinstitute/dsde/workbench/avram/api/AvramRoutes.scala

```diff
@@ -46,6 +66,12 @@ class AvramRoutes {
     Await.result(result, Duration.Inf)
   }
 
+  private def getToken(req: HttpServletRequest) = {
```

Comment:
Oops... this is unused. I meant to remove this.

Reply:
+1 removing